### ISSUE TITLE
Port 3306 connection issue

### ISSUE THREAD

**Issue by Thinking80s** [apitable/apitable#1723]
## Port 3306 connection is closed

**Comment by Thinking80s** [comment]
docker deploy use 
`host.docker.internal`
replace 127.0.0.1 or localhost

[https://stackoverflow.com/questions/48546124/what-is-the-linux-equivalent-of-host-docker-internal](url)

**Comment by mr-kelly** [comment]
Yes, as mentioned by the community, when connecting from inside a Docker container to a MySQL database running on the host machine, you should use `host.docker.internal` instead of `localhost` or `127.0.0.1`.
